python/py-allennlp/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        allenai allennlp 2.0.1 v
+revision            0
+name                py-${github.project}
+categories-append   textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         An open-source NLP research library, built on PyTorch.
+
+long_description    An Apache 2.0 NLP research library, built on \
+                    PyTorch, for developing state-of-the-art deep \
+                    learning models on a wide variety of linguistic \
+                    tasks.
+
+homepage            https://www.allennlp.org/
+
+checksums           rmd160  1c2c31e6f40daf34a3dd83ec225bc98049716f76 \
+                    sha256  17f8920d39407fea1731eeca7e6ad4c0bfba9808e13aee469e48f12031413b35 \
+                    size    2667174
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-boto3 \
+                    port:py${python.version}-conllu \
+                    port:py${python.version}-editdistance \
+                    port:py${python.version}-filelock \
+                    port:py${python.version}-flaky \
+                    port:py${python.version}-flask \
+                    port:py${python.version}-flask-cors \
+                    port:py${python.version}-ftfy \
+                    port:py${python.version}-gevent \
+                    port:py${python.version}-h5py \
+                    port:py${python.version}-jsonnet \
+                    port:py${python.version}-jsonpickle \
+                    port:py${python.version}-matplotlib \
+                    port:py${python.version}-nltk \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-numpydoc \
+                    port:py${python.version}-overrides \
+                    port:py${python.version}-parsimonious \
+                    port:py${python.version}-pytorch \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-responses \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-scikit-learn \
+                    port:py${python.version}-scipy \
```

Comment:
per lint, `port:py${python.version}-scipy` appears twice here.

Reply:
Fixed, thanks, along with a bunch of other dependencies that have changes since this PR was posted originally.